src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java

```diff
@@ -6,7 +6,7 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Keep only reads whose strand is forward or reverse
+ * Keep only reads whose strand is as specified.
```

Comment:
It is not longer mentioning reverse/forward, but it refers to the SAM 0x10 flag (reverse complement sequence).